src/components/onboarding/survey/hooks/useSurveyState.tsx

```
@@ -86,16 +88,33 @@ export const useSurveyState = () => {
       case 0: // Basic Information (or OAuth Welcome)
         return formData.firstName.trim() !== "" && formData.lastName.trim() !== "";
       
-      case 1: // Contact/Auth Information  
+      case 1: // Contact/Auth Information & Address
+        // Check email and password requirements
+        let emailPasswordValid = false;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)